frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -255,3 +255,46 @@ Feature: Pipeline Runs
              Then user will see the pipeline run name with failed status
               And user will see failure message below pipeline runs
               And user will also see the log snippet
+
+
+        @regression
+        Scenario Outline: Start pipeline modal with PVC Workspaces
+            Given pipeline "pipeline-workspace" with PVC "git-PVC" workspace is present
+             When user selects "Start" option from kebab menu for pipeline "pipeline-workspace"
+              And user navigates to Workspaces section
+              And user clicks on dropdown with PVC selected by default to check options
+              And user clicks on dropdown with PVC names
+             Then user is able to see the PVC "git-PVC" and "create PVC" option present
+
+
+        @regression
+        Scenario Outline: Start pipeline modal with no Workspaces
+            Given pipeline "pipeline-no-workspace" with no workspace is present
+             When user selects "Start" option from kebab menu for pipeline "pipeline-no-workspace"
+              And user navigates to Workspaces section
+             Then user can see on dropdown with PVC selected by default
+              And user can see Create PVC selected
```

Comment:
Confused about this test.

Your Pipeline has NO workspaces? If that's true, then you will not have a `Workspaces section` to navigate to... the Start Modal is based off of what you have in your params, resources, and workspaces... if you don't have a workspace, you won't get the section.

Reply:
I took reference from the doc https://docs.google.com/document/d/13JTQJ_zKHbxKUCudSxlXJ29xZ9CJLC5Cv1-_b8xZYgk/edit
There it is mentioned in case of no defined Workspace the Create PVC will be selected and we can create a PVC workspace

Reply:
Ah interesting... there was an original direction to have Create PVC try to be smart to select a previous PVC instead of jumping to the Create section... but your wording is incorrect I think. "Start pipeline modal with no Workspaces"? I think you mean "Start pipeline modal with no PVCs created" ... because if you have no workspaces (which can be PVCs, ConfigMaps, Secrets OR -- coming soon -- volumeClaimTemplates) you do not have a workspace section.